OmiseSDK/PaymentCreatorController.swift

```diff
@@ -535,6 +535,7 @@ extension PaymentCreatorController {
         .internetBankingSCB,
         .internetBankingBBL,
         .mobileBankingSCB,
+        .mobileBankingOCBCPAO,
```

Comment:
Is OCBC PAO not a payment method for Singapore? Then we should add it to the presets for Singapore.